src/tbb/dynamic_link.h

```diff
@@ -71,8 +71,9 @@ using dynamic_link_handle = void*;
 const int DYNAMIC_LINK_GLOBAL        = 0x01;
 const int DYNAMIC_LINK_LOAD          = 0x02;
 const int DYNAMIC_LINK_WEAK          = 0x04;
+const int DYNAMIC_LINK_LOCAL         = 0x08;
 
-const int DYNAMIC_LINK_LOCAL_BINDING = 0x08 | DYNAMIC_LINK_LOAD;
+const int DYNAMIC_LINK_LOCAL_BINDING = DYNAMIC_LINK_LOCAL | DYNAMIC_LINK_LOAD;
```

Comment:
It seems that this is not used

Reply:
It used here: https://github.com/oneapi-src/oneTBB/blob/master/src/tbb/governor.cpp#L393